plotting/colormap.py

```diff
@@ -287,11 +287,11 @@ def find_colormap(name):
     "potential sub surface channel": mcolors.ListedColormap(["#ecf0f1", "#f57732"]),
     "thermal": cmocean.cm.thermal,
     "neo_sst": mcolors.ListedColormap(
-        np.loadtxt(os.path.join(data_dir, "neo_sst.txt"))
+        np.loadtxt(Path(data_dir).joinpath("neo_sst.txt"))    
     ),
-    "BuYlRd": mcolors.ListedColormap(np.loadtxt(os.path.join(data_dir, "BuYlRd.txt"))),
+    "BuYlRd": mcolors.ListedColormap(np.loadtxt(Path(data_dir).joinpath("BuYlRd.txt"))),
     "temperature": mcolors.ListedColormap(
-        np.loadtxt(os.path.join(data_dir, "temperature.txt"))
+        np.loadtxt(Path(data_dir).joinpath("temperature.txt"))
```

Comment:
Just use `data_dir.joinpath()` here instead of calling `Path()` on data_dir again.